pdns/dnsdistdist/Makefile.am

```diff
@@ -191,19 +191,19 @@ if HAVE_RE2
 dnsdist_LDADD += $(RE2_LIBS)
 endif
 
+if HAVE_LIBSSL
+dnsdist_LDADD += $(LIBSSL_LIBS)
```

Comment:
The addition of `$(LIBSSL_CFLAGS)` to `AM_CPPFLAGS` is still only done when either `HAVE_DNS_OVER_TLS` or `HAVE_DNS_OVER_HTTPS` is set, perhaps we should change that? Otherwise we might include the wrong files.

Reply:
HAVE_LIBSSL cannot be true if neither is set (enforced in the configure.ac)

Reply:
You are right so it doesn't matter!